src/components/AvatarGroup/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { BaseProps } from '../types';
+import { StyledContainer, StyledAvatarContainer } from './styled';
+import RenderIf from '../RenderIf';
+import Avatars from './Avatars';
+import Counter from './Counter';
+import { AvatarSizes, AvatarProps } from './types';
+
+interface AvatarGroupProps extends BaseProps {
+    size?: AvatarSizes;
+    avatars?: Array<AvatarProps> | any;
```

Comment:
we should not use `any` since this will allow users to pass any other data type, the purpose of this is to force avatars to be an array